addon/components/discover-page/template.hbs

```diff
@@ -0,0 +1,180 @@
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+    </div>
+</div>
+<div class="search-main p-v-xl">
+    <div class="container">
+        <div class="row search-results-section">
+            <div class="col-sm-4 col-xs-12 search-sidebar"> {{!LEFT PANEL - SEARCH FACETS}}
+                {{! ACTIVE FILTERS - FOR PREPRINTS, REGISTRIES}}
+                {{#if showActiveFilters}}
+                    <div id="activeFilters" class="well well-sm p-md">
+                        <div class="row">
+                            <div class="col-sm-6">
+                                <h4>{{t "eosf.components.discoverPage.activeFilters.heading"}}:</h4>
+                            </div>
+                            <div class="col-sm-6">
+                                <span class="pull-right">
+                                    {{!CLEAR FILTERS BUTTON}}
+                                    <button type="button" class="btn btn-default btn-sm" {{action "clearFilters"}}>{{clearFiltersButton}} {{fa-icon "remove"}}</button>
+                                </span>
+                            </div>
+                        </div>
+                        <div id="activeFiltersList">
+                            {{#each activeFilters.providers as |filter|}}
+                                <span class='preprint-filter provider-filter'>
+                                    {{filter-replace filter filterReplace}}
+                                    {{#unless theme.isProvider}}
+                                        {{!ACTIVE PROVIDER FILTERS}}
+                                        <span class='pointer'>{{fa-icon 'times-circle' click=(action 'updateFilters' 'providers' filter)}}</span>
+                                    {{/unless}}
+                                </span>
+                            {{/each}}
+                            {{#each activeFilters.subjects as |filter|}}
+                                {{!ACTIVE SUBJECT FILTERS}}
+                                <span class='preprint-filter subject-filter'>{{filter}} <span class='pointer'>{{fa-icon 'times-circle' click=(action 'updateFilters' 'subjects' filter)}}</span></span>
+                            {{/each}}
+                            {{!ACTIVE TYPE FILTERS}}
+                            {{#each activeFilters.types as |filter|}}
+                                <span class='preprint-filter type-filter'>{{filter}} <span class='pointer'>{{fa-icon 'times-circle' click=(action 'updateFilters' 'types' filter)}}</span></span>
+                            {{/each}}
+                        </div>
+                    </div>
+                {{/if}}
+
+                {{!SEARCH FACETS}}
+                {{#if showActiveFilters}}
+                    <h4> {{t 'eosf.components.discoverPage.refineSearch'}} </h4>
+                {{/if}}
+                {{faceted-search
+                    onChange='filtersChanged'
```

Comment:
If the network request is a 400, we show Lucene Search Help.  If a network request fails for any other reason, they get a message saying that Search is Unavailable.  I don't know if I agree that the search-facet components should be reset.  Several times as I was developing this, Staging SHARE would be down for a few seconds and my search would fail. I could just retry the search because my filters were still there. Regardless, Preprints and Registries are not using these actions right now, these are for the Ember-SHARE facets.